tests/specs/signedin-test.js

```diff
@@ -37,24 +28,15 @@ test('returns correct signedIn state', function (t) {
     url: baseURL
   })
 
-  var accountWithProfile = merge(options, {profile: profile})
-  var returnedAccount = {
-    username: options.username,
-    session: {
-      id: 'sessionid123'
-    },
-    profile: profile
-  }
-
   nock(baseURL)
     .put('/session/account')
-    .reply(200, JSON.stringify(returnedAccount))
+    .reply(200, JSON.stringify(signUpResponse))
     .put('/session')
-    .reply(201, JSON.stringify(returnedAccount.session))
+    .reply(201, JSON.stringify(signInResponse))
     .delete('/session')
     .reply(204)
 
-  account.signUp(accountWithProfile)
+  account.signUp(options)
```

Comment:
I thought signUp should call signIn internally now, but I'd say let’s make a follow up issue for that